torch_xla/csrc/xla_custom_call.cpp

```diff
@@ -0,0 +1,22 @@
+#include "torch_xla/csrc/xla_custom_call.h"
+#include <iostream> 
+
+
+// TODO: number of inputs and shapes of inputs and outputs are
+// hardcoded. Need to be sent in the args. 
+// TODO: template the datatype?
+void argmin_custom(void* out, const void** in) {
```

Comment:
dumb question, this only works for GPU right? since it assumes memory layout.

Reply:
yes, the idea is GPUs for this work

Reply:
And that particular example is only for CPU